premake5.lua

```diff
@@ -510,35 +510,56 @@ if isTargetWindows then
         
         systemversion "10.0.14393.0"
 
-        -- For Windows targets, we want to copy d3dcompiler_47.dll,
+        removefiles { "tools/render-test/cpu-render-test-main.cpp" }
```

Comment:
It might look a bit cleaner if the cpu-variant-specific files went in `tools/render-test/cpu/*` (and similarly for the gpu variant) and were explicitly included in the cpu variant's build rather than being excluded from the build for the gpu variant.